debian/postinst.d/11-user-creator.sh

```diff
@@ -12,7 +12,7 @@ main() {
     IFS=','
 
     local group_name=(${SERVICE_GROUP_NAMES})
-    adduser --system --no-create-home --ingroup ${group_name} ${SERVICE_USER_NAME}
+    adduser --system --no-create-home --shell /bin/bash --ingroup ${group_name} ${SERVICE_USER_NAME}
```

Comment:
for system users or services we basically do not want them to be able to login. so we either use `/bin/nologin` or `/bin/false` as their shell. 
![image](https://user-images.githubusercontent.com/95352874/236663033-810b7eda-7372-4367-81f5-0b44a6562e4f.png)

why do you specify the shell for keepalived ?

Reply:
301a886b2aedc66845c6e3e9995990e45223861f